types/artifacts.d.ts

```diff
@@ -317,6 +317,7 @@ declare global {
       export interface AnchorElement {
         rel: string
         href: string
+        rawHref: string
```

Comment:
A comment on why this might differ would be useful.

Reply:
True, have done this:

```ts
/** The computed href property: https://www.w3.org/TR/DOM-Level-2-HTML/html.html#ID-88517319, use `rawHref` for the exact attribute value */
href: string
/** The exact value of the href attribute value, as it is in the DOM */
rawHref: string

```